src/Login/Login.test.tsx

```diff
@@ -5,7 +5,7 @@ import { render, fireEvent, waitFor } from '@testing-library/react';
 import { MemoryRouter } from 'react-router-dom';
 // window.MutationObserver = require("mutation-observer");
 import MutationObserver from '@sheerun/mutationobserver-shim';
-window.MutationObserver = MutationObserver ;
+window.MutationObserver = MutationObserver;
```

Comment:
Did this resolve an error - like the one Léah and I were attempting to fix this AM?

Reply:
I'm not sure, didn't try it out. I'm not sure why GH is even saying this is different because these lines of code look the same, no?